client/src/main/java/org/spine3/base/MapStringifier.java

```diff
@@ -0,0 +1,175 @@
+ * should be registered in the {@code StringifierRegistry} class
+ * for the correct usage of {@code MapStringifier}.
+ *
+ * <h3>Example</h3>
+ *
+ * {@code
+ *  // The registration of the stringifier.
+ *  final Type type = new TypeToken<Map<String, Long>>(){}.getType();
+ *  StringifierRegistry.getInstance().register(stringifier, type);
+ *
+ *  // Obtain already registered `MapStringifier`.
+ *  final Stringifier<Map<String, Long>> mapStringifier = StringifierRegistry.getInstance()
+ *                                                                           .getStringifier(type);
+ *
+ *  // Convert to string.
+ *  final Map<String, Long> mapToConvert = ...
+ *  final String convertedString = mapStringifier.toString(mapToConvert);
+ *
+ *  // Convert from string.
+ *  final String stringToConvert = ...
+ *  final Map<String, Long> convertedMap = mapStringifier.fromString(stringToConvert);
+ * }
+ *
+ * @param <K> the type of the keys in the map
+ * @param <V> the type of the values in the map
+ */
+class MapStringifier<K, V> extends Stringifier<Map<K, V>> {
+
+    private static final char DEFAULT_ELEMENT_DELIMITER = ',';
+    private static final String ESCAPE_SEQUENCE = "\\";
+    private static final String KEY_VALUE_DELIMITER = ESCAPE_SEQUENCE + ':';
+
+    /**
+     * The delimiter for the passed elements in the {@code String} representation,
+     * {@code DEFAULT_ELEMENT_DELIMITER} by default.
+     */
+    private final String delimiter;
+    private final Class<K> keyClass;
+    private final Class<V> valueClass;
+
+    MapStringifier(Class<K> keyClass, Class<V> valueClass) {
+        super();
+        this.keyClass = keyClass;
+        this.valueClass = valueClass;
+        this.delimiter = ESCAPE_SEQUENCE + DEFAULT_ELEMENT_DELIMITER;
+    }
+
+    /**
+     * That constructor should be used when need to use
+     * a custom delimiter of the elements during conversion.
+     *
+     * @param keyClass   the class of the key elements
+     * @param valueClass the class of the value elements
+     * @param delimiter  the delimiter for the passed elements via string
+     */
+    MapStringifier(Class<K> keyClass, Class<V> valueClass, String delimiter) {
+        super();
+        this.keyClass = keyClass;
+        this.valueClass = valueClass;
+        this.delimiter = ESCAPE_SEQUENCE + delimiter;
+    }
+
+    @Override
+    protected String toString(Map<K, V> obj) {
+        final String result = obj.toString();
+        return result;
+    }
+
+    @Override
+    protected Map<K, V> fromString(String s) {
+        final String[] buckets = s.split(Pattern.quote(delimiter));
+        final Map<K, V> resultMap = newHashMap();
+
+        for (String bucket : buckets) {
+            saveConvertedBucket(resultMap, bucket);
+        }
+        Ints.stringConverter();
+        return resultMap;
+    }
+
+    private Map<K, V> saveConvertedBucket(Map<K, V> resultMap, String element) {
```

Comment:
I think, this method signature should be reviewed. Right now its name and its arguments (including their order) do not tell what this method is for.